webknossos/webknossos/dataset/layer/view/_array.py

```diff
@@ -797,6 +815,7 @@ def __setstate__(self, d: dict[str, Any]) -> None:
 
 class Zarr3Array(TensorStoreArray):
     data_format = DataFormat.Zarr3
+    _ts_data_format = "zarr3"
```

Comment:
I’d rename it to _tensorstore_data_format to make it more glanceable what this is about. Or maybe even create a translation function somewhere from DataFormat to the tensorstore string and use that instead of a defining this redundantly here.